packages/stratimikos/cmake/Dependencies.cmake

```diff
@@ -2,9 +2,15 @@
 # adapters will get enabled by default.  Note that Belos and ML do *not* have
 # a required dependence on Epetra but the Stratimikos Belos and ML adapters
 # need the Thyra/Epetra adapters.
-SET(LIB_REQUIRED_DEP_PACKAGES ThyraEpetraAdapters)
-SET(LIB_OPTIONAL_DEP_PACKAGES Amesos AztecOO Belos Ifpack ML EpetraExt)
-SET(TEST_REQUIRED_DEP_PACKAGES)
+IF (Stratimikos_MAKE_EPETRA_OPTIONAL)
```

Comment:
Why not just make Epetra optional and take out this if statement (i.e. hard-code Stratimikos_MAKE_EPETRA_OPTIONAL=TRUE)?  Yes, this would technically break backward compatibility but the default configure for will enable Epetra and therefore the actually code and tests will not change.  I would be comfortable with removing Epetra as a required dependency without requiring someone to set `-DStratimikos_MAKE_EPETRA_OPTIONAL=TRUE`.  If people are properly using black-listing instead of white-listing, then this will not break anyone's configure and build of Trilinos.